packages/trader/src/Modules/Trading/Components/Elements/purchase-fieldset.tsx

```diff
@@ -107,7 +132,7 @@ const PurchaseFieldset = ({
                         }}
                         onMouseLeave={() => {
                             if (!is_disabled) {
-                                onHoverPurchase(false);
+                                onHoverPurchase(false, type);
```

Comment:
@kate-deriv nit: according to updateBarrierShade() signature called inside onHoverPurchase(): when it's false, type doesn't matter :) You might wanna remove it. Since useTraderStores is used in this PR, onHoverPurchase type can be overriden there. Well, it's gonna work with or without it, and it was approved earliie, so it's up to you. :)